dizoo/atari/config/serial/Asterix/asterix_mdqn_config.py

```diff
@@ -0,0 +1,66 @@
+from copy import deepcopy
+from easydict import EasyDict
+
+asterix_mdqn_config = dict(
+    exp_name='asterix_mdqn_seed0',
+    env=dict(
+        collector_env_num=8,
+        evaluator_env_num=8,
+        n_evaluator_episode=8,
+        stop_value=20000,
+        env_id='Asterix-v0',
+        #'ALE/SpaceInvaders-v5' is available. But special setting is needed after gym make.
+        frame_stack=4,
+        manager=dict(shared_memory=False, ),
+    ),
+    policy=dict(
+        cuda=True,
+        priority=False,
+        model=dict(
+            obs_shape=[4, 84, 84],
+            action_shape=9,
+            encoder_hidden_size_list=[32, 32, 64, 512],
+        ),
+        nstep=1,
+        discount_factor=0.99,
+        entropy_tau=0.03,
+        m_alpha=0.9,
+        learn=dict(
+            update_per_collect=4,
+            batch_size=32,
+            learning_rate=0.00005,
+            target_update_freq=8,
+            learner=dict(train_iterations=int(3e7), hook=dict(save_ckpt_after_iter=1000000, ))
```

Comment:
remove `train_iterations` in config, instead of indicating it in the argument of serial_pipeline (max_train_iter)